object_management:
  publish_frequency: 5
  switch_period: 1.0
  img_topic_sources: ["/kinect/color/image_raw","/camera/color/image_raw"]